content/en/account_management/plan_and_usage/cost_details.md

```diff
@@ -31,6 +27,12 @@ To view the Estimated Cost Summary and Cost Chargebacks data, you must be a Data
 
 Alternately, roles with Billing Read (`billing_read`) and Usage Read (`usage_read`) [permissions][1] can view the Estimated Cost Summary and Cost Chargebacks data.
 
+### Feature Access
+
+[Estimated Cost Summary (parent organization)][7] and [Cost Chargebacks][8] are available out-of-the-box if your organization meets the feature criteria.
+
+[Estimated Cost Summary (sub-organization)][9] is in beta. To request access and confirm your organization meets the feature criteria, contact your account representative or [Support][6].
```

Comment:
Optional: We usually highlight this in other docs with either a {{< callout >}} or <div class="alert alert-info"> which is the blue box. If that's something you'd like helping formatting, let me know.

Reply:
Thanks @estherk15 - is this what you're referring to? https://a.cl.ly/QwuwjGXg

is the "blue box" something different? if we move forward with the callout, would you recommend we put that at the top where it currently is in production today?

Reply:
If this is something that you anticipate customers asking about, I recommend putting it at the top to avoid confusion. The image you have is the callout, the blue box looks like this: https://a.cl.ly/nOuL1N6G
![image](https://github.com/DataDog/documentation/assets/41168354/745074e4-1e97-4e10-947c-a33389dc0e3c)